flexget/plugins/clients/qbittorrent.py

```diff
@@ -152,6 +154,10 @@ def add_entries(self, task, config):
                 form_data['label'] = label  # qBittorrent v3.3.3-
                 form_data['category'] = label  # qBittorrent v3.3.4+
 
+            add_paused = entry.get('paused', config.get('paused'))
```

Comment:
Shouldn't these both be `add_paused` rather than just `paused`?

Reply:
You're right, thanks!